specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-08-08-preview/AlertsManagement.json

```diff
@@ -1,117 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "version": "2021-08-08-preview",
-    "title": "Azure Alerts Management Service Resource Provider",
-    "description": "Azure Alerts Management Service provides a single pane of glass of alerts across Azure Monitor."
-  },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "flow": "implicit",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
-      }
-    }
-  },
-  "paths": {
-    "/providers/Microsoft.AlertsManagement/operations": {
```

Comment:
Why do you remove this path in new version? I don't see you copy previous paths to this version.

Reply:
@raych1 AlertsManagement.json in the 2019-05-05-preview version contains both the operations API, and alerts APIs. In this version we only change the "alerts processing rule" APIs. So we referenced the 2019 version for the alerts and operation APIs (2019-05-05-preview/AlertsManagement.json) under the package-preview-2021-08.